batch7crud-roll3/src/main/java/com/lftechnology/batch7crud/db/DBConnection.java

```diff
@@ -0,0 +1,30 @@
+package com.lftechnology.batch7crud.db;
+
+import java.sql.SQLException;
+import javax.naming.Context;
+import javax.naming.InitialContext;
+import javax.naming.NamingException;
+import javax.sql.DataSource;
+import com.lftechnology.batch7crud.exception.DataException;
+import java.sql.Connection;
+
+public class DBConnection {
+	public static Connection getConnection() throws DataException {
+		try {
+			Context initCtx = new InitialContext();
+
+			Context envCtx = (Context) initCtx.lookup("java:comp/env");
+
+			DataSource ds = (DataSource)
+
+			envCtx.lookup("jdbc/students_info");
+
+			Connection conn = ds.getConnection();
+			return conn;
+		} catch (SQLException e) {
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1166)